.github/workflows/hugo.yml

```diff
@@ -0,0 +1,74 @@
+# Sample workflow for building and deploying a Hugo site to GitHub Pages
+name: Deploy Hugo site to Pages
+
+on:
```

Comment:
I would like to also build the site daily by using a scheduled task.
https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule

But it something we ca do on a separate PR.